frontend/src/component/layout/MainLayout/NavigationSidebar/IconRenderer.tsx

```diff
@@ -29,10 +30,14 @@ import GitHubIcon from '@mui/icons-material/GitHub';
 import LibraryBooksIcon from '@mui/icons-material/LibraryBooks';
 import FactCheckOutlinedIcon from '@mui/icons-material/FactCheckOutlined';
 import PersonalDashboardIcon from '@mui/icons-material/DashboardOutlined';
-import type { FC } from 'react';
+import { ProjectIcon } from 'component/common/ProjectIcon/ProjectIcon';
+import PlaygroundIcon from '@mui/icons-material/AutoFixNormal';
 
 // TODO: move to routes
-const icons: Record<string, typeof SvgIcon> = {
+const icons: Record<
+    string,
+    typeof SvgIcon | FC<ComponentProps<typeof SvgIcon>>
```

Comment:
What's this signature change about?

Reply:
Project icon is a different type, because it's a custom SVG, not from MUI. This probably could be aligned, but is not worth the time.